recipes/recipes_emscripten/rustworkx/recipe.yaml

```diff
@@ -0,0 +1,41 @@
+context:
+  name: rustworkx
+  version: 0.17.0a3
+
+package:
+  name: ${{name}}
+  version: ${{ version }}
+
+source:
+- url: https://github.com/IvanIsCoding/rustworkx/releases/download/v0.17.0a3/rustworkx-0.17.0a3.tar.gz
+  sha256: 8bd0c295134e2b0c03808d4e69428b41153849db6488839084a78793c337f191
+
+build:
+  number: 0
+
+requirements:
+  build:
+  - cross-python_${{target_platform}}
+  - cffi
+  - setuptools-rust
+  - rust
+  - rust-src
+  - maturin
+
+  host:
+  - python
+  - openssl
+  - cffi
+  run:
+  - cffi
```

Comment:
I copied this from the other Rust recipes. I am not sure it is necessary, but the tests passed so I am not going to touch it.

Reply:
The compiler package should be added to the build dependencies, otherwise we might not get the emscripten abi package